_data/authors.yml

```diff
@@ -9,4 +9,10 @@ keunes:
   image: keunes.png
   email:
   web: https://github.com/keunes
-  position: marketing
\ No newline at end of file
+  position: marketing
+loucasal:
+  name: Loucasal
+  image: loucasal.png
```

Comment:
@loucasal any preferred user icon instead of the GitHub one?

Reply:
This one is fine - thanks!